go_modules/spec/dependabot/go_modules/update_checker/latest_version_finder_spec.rb

```diff
@@ -144,16 +144,19 @@ module foobar
     context "for a Git pseudo-version with releases available" do
       let(:dependency_version) { "1.0.0-20181018214848-ab544413d0d3" }
 
-      it "doesn't return the releases, currently" do
-        expect(finder.latest_version).to eq(dependency_version)
+      it "returns the latest pre-release" do
```

Comment:
Could we add a test where it upgrades to a release (`1.2.0`)? I think that will be a common use case when an older dependency transitions to using modules.

Reply:
I added some more realistic tests for this case, both version updates and security updates, based on golang.org/x/net.